tests/playwright/sanity/container.test.js

```diff
@@ -75,7 +76,7 @@ test.describe( 'Container tests', () => {
 
 		await editor.hideVideoControls();
 		await editor.togglePreviewMode();
-		await page.waitForLoadState( 'domcontentloaded' );
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
What is advantage of the `networkidle` method?

I had added this waiting method, because the video and google maps widgets in some cases would re-render after a style change.